### ISSUE TITLE
Upgrade FINOS App Directory Apps to use FDC3 for the web.

### ISSUE THREAD

**Issue by robmoffat** [finos/FDC3-Sail#163]

**Comment by SeeWhatsOn** [comment]
Does this align with the example apps @robmoffat ? Also I vote for keeping the App Directory.

**Comment by robmoffat** [comment]
no, this is a different thing.  Although really there's only AdapTable in the App Directory that needs to update.  Everything else is done

**Comment by SeeWhatsOn** [comment]
Put this ticket with the example apps (FDC3 repo)

**Comment by SeeWhatsOn** [comment]
Not a sail issue and needs to be raised in the FDC3 meetings - create an issue @SeeWhatsOn on FDC3
